adafruit_debouncer.py

```diff
@@ -62,7 +63,7 @@ def __init__(self, io_or_predicate, interval=0.010):
            :param int interval: bounce threshold in seconds (default is 0.010, i.e. 10 milliseconds)
         """
         self.state = 0x00
-        if isinstance(io_or_predicate, digitalio.DigitalInOut):
+        if isinstance(io_or_predicate, (digitalio.DigitalInOut, touchio.TouchIn)):
```

Comment:
could this be changed to 'if hasattr(io_or_predicate, 'value')` ?  This would mean that the import of touchio was not needed, but would also make it work with other io types, such as on a seesaw.